r276494 from upstream HEAD

Don't include bsd.sys.mk within bsd.kmod.mk, else CFLAGS will be given
twice on the command line.  That would override CWARNFLAGS, and causes
some modules to fail to build under Clang.

--- a/src/share/mk/bsd.kmod.mk
+++ b/src/share/mk/bsd.kmod.mk
@@ -15,5 +15,3 @@
 .endif
 
 .include "${SYSDIR}/conf/kmod.mk"
-
-.include <bsd.sys.mk>
